packages/hardhat-test/test/hardhat.test.ts

```diff
@@ -21,7 +21,7 @@ describe('TypeChain x Hardhat', () => {
 
     const counterFactory1 = new Counter__factory()
     const counterFactory2 = new Counter__factory(deployer)
-    const counterFactory3 = new Counter__factory(artifact.abi, artifact.bytecode)
-    const counterFactory4 = new Counter__factory(artifact.abi, artifact.bytecode, deployer)
+    // const counterFactory3 = new Counter__factory(artifact.abi, artifact.bytecode)
```

Comment:
@fvictorio  have a look at changes in this file

Reply:
I think you can uncomment these two lines and they will compile if you change this import:

```ts
import type { Interface } from '@ethersproject/abi'
```

to this:

```ts
import type { Interface } from 'ethers'
```

Reply:
Magic